packages/app-desktop/gui/NoteList/NoteList.tsx

```diff
@@ -340,11 +340,39 @@ const NoteListComponent = (props: Props) => {
 		return noteIndex;
 	};
 
+	const noteItem_noteMove = async (keyCode: any) => {
+		if (props.notesParentType !== 'Folder') return;
+		if (props.noteSortOrder !== 'order') {
+			const doIt = await bridge().showConfirmMessageBox(_('To manually sort the notes, the sort order must be changed to "%s" in the menu "%s" > "%s"', _('Custom order'), _('View'), _('Sort notes by')), {
+				buttons: [_('Do it now'), _('Cancel')],
+			});
+			if (!doIt) return;
+
+			Setting.setValue('notes.sortOrder.field', 'order');
+			return;
+		}
+		const noteIds = props.selectedNoteIds;
+		if ((keyCode === 40)) {
+			const noteId = noteIds[0];
+			const targetNoteIndex = BaseModel.modelIndexById(props.notes, noteId) + 2;
+			void Note.insertNotesAt(props.selectedFolderId, noteIds, targetNoteIndex);
+		}
+
+		if ((keyCode === 38)) {
+			const noteId = noteIds[0];
+			const targetNoteIndex = BaseModel.modelIndexById(props.notes, noteId) - 1;
+			void Note.insertNotesAt(props.selectedFolderId, noteIds, targetNoteIndex);
+		}
```

Comment:
Refactor to reduce copied and pasted code

Reply:
Have refactored the code that is repeated.